edit.php

```diff
@@ -142,7 +142,11 @@
 	}
 }
 
-
+$crumbs = generate_crumbs(array(
+	array($lang_common['Index'], 'index.php'),
+	array($cur_post['forum_name'], 'viewforum.php?id='.$cur_post['fid']),
+	array($cur_post['subject'], 'viewtopic.php?id='.$cur_post['tid']),
+	$lang_post['Edit post']));
```

Comment:
For all of these, please move the closing paren to an extra line, and add a trailing comma to the last array element.